components/tyk-gateway/templates/deployment-gw-repset.yaml

```
@@ -231,11 +231,11 @@ spec:
             value: "/mnt/tyk-gateway/policies/policies.json"
         {{- end}}
           - name: TYK_GW_HASHKEYS
-            {{- if .Values.global.hashKeys }}
-            value: {{ .Values.global.hashKeys | toString | quote }}
-            {{- else }}
-            value: {{ .Values.gateway.hashKeys | toString | quote }}
-            {{- end }}
+          {{- if eq .Values.global.hashKeys nil }}
+            value: {{ .Values.gateway.hashKeys | quote }}
+          {{- else }}
+            value: {{ .Values.global.hashKeys | quote }}
+          {{- end }}
```

No need to check for `.Values.gateway.hashKeys` because it does not exist
```suggestion
            value: {{ .Values.global.hashKeys | quote }}
```